src/Microsoft.Data.Sqlite/SqliteConnection.cs

```diff
@@ -78,10 +78,10 @@ public override void Open()
                 throw new InvalidOperationException(Strings.OpenRequiresSetConnectionString);
             }
 
-            var rc = NativeMethods.sqlite3_enable_shared_cache(ConnectionStringBuilder.CacheMode == CacheMode.Shared); // called to ensure mode set correctly before a connection is created
-            MarshalEx.ThrowExceptionForRC(rc, _db);
+            var flags = Constants.SQLITE_OPEN_READWRITE | Constants.SQLITE_OPEN_CREATE;
```

Comment:
It's the default behavior of sqlite3_open and sqlite3_open16, but https://sqlite.org/c3ref/open.html doesn't seem clear to me about what happens if I pass in 0 for the flags parameter. I figure it's safer to be explicit in defining the behavior we want.